tests/test_meshtastic_utils.py

```
@@ -2260,5 +2260,286 @@ def test_check_connection_uncovered_paths(self, mock_logger):
+                # Should still return result with firmware version parsed
+                self.assertTrue(result["success"])
+                self.assertEqual(result["firmware_version"], "2.3.15")
+                # Verify timeout was logged
+                mock_logger.debug.assert_called_with(
+                    "getMetadata() timed out after 30 seconds"
+                )
+
+    @patch("mmrelay.meshtastic_utils.logger")
+    @patch("bleak.BleakClient")
+    def test_disconnect_ble_by_address_is_connected_bool_true(
+        self, mock_bleak, mock_logger
```

Ruff (0.14.11)

Unused method argument: `mock_logger`

(ARG002)